### PR TITLE
Adding expander tests

### PR DESCRIPTION
**Pull request by lawrencegripper**
This adds a set of integration tests for the core set of expanders

- [x] Tenant Epxander
- [x] Subscription Expander
- [ ] Resource Group Expander

### CODE REVIEW

File: internal/pkg/expanders/integration_test.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	"gopkg.in/h2non/gock.v1"
+)
+
+func Test_Expanders(t *testing.T) {
+
+	InitializeExpanders(nil)
+	expanders := getRegisteredExpanders()
+
+	// Add special case of default expander
+	expanders = append(expanders, &DefaultExpander{})
```
Comment (lawrencegripper) [inline internal/pkg/expanders/integration_test.go]: @stuartleeks and @jjcollinge what do you think of this approach?

Comment (lawrencegripper) [reply internal/pkg/expanders/integration_test.go]: (this section of file not this line in particular)

File: internal/pkg/expanders/testdata/armsamples/resource/failingResource.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "id": "/subscriptions/5774ad8f-0000-0000-0000-0447910568d3/resourceGroups/stable/providers/Microsoft.ContainerRegistry/registries/test",
```
Comment (jjcollinge) [inline internal/pkg/expanders/testdata/armsamples/resource/failingResource.json]: should these IDs be in?

Comment (lawrencegripper) [reply internal/pkg/expanders/testdata/armsamples/resource/failingResource.json]: Fixed thanks

**Pull request merged**